stetho/src/main/java/com/facebook/stetho/inspector/MethodDispatcher.java

```diff
@@ -63,6 +67,13 @@ public JSONObject dispatch(JsonRpcPeer peer, String methodName, @Nullable JSONOb
           "Not implemented: " + methodName,
           null /* data */));
     }
+
+    if (mAnalyticsLogger != null) {
+      mAnalyticsLogger.track(
+          "inspector_method_called",
```

Comment:
Again I'm not convinced this strategy will yield the result you want.  Can you please try to model the kind of data this will produce and explain how it will be used to evaluate usage?  To me, I think it'll just be a big awkward blob of data that will mislead as much as it informs (for instance, it would be easy to assume Network.* methods being invoked correspond to the network tab being used but they don't)